docker/Dockerfile

```diff
@@ -1,27 +1,35 @@
-FROM alpine:3.10
+FROM alpine:3.11 AS downloader
 
-ENV APP_VERSION=5.9.0
-ENV PATH="/mattermost-push-proxy/bin:${PATH}"
-
-# Install some needed packages
-RUN apk add --no-cache\
-      netcat-openbsd \
-      ca-certificates
+ENV MM_PROXY_PACKAGE=https://github.com/mattermost/mattermost-push-proxy/releases/download/v5.21/mattermost-push-proxy.tar.gz
```

Comment:
Wondering if it makes sense to pass v5.21 as an env var ?

Reply:
done, was a mistake should be ARG